include/z64.h

```diff
@@ -755,6 +755,12 @@ typedef struct OverlayRelocationSection {
     /* 0x14 */ u32 relocations[1]; // size is nRelocations
 } OverlayRelocationSection; // size >= 0x18
 
+// This struct is used at osAppNMIBuffer which is not at an 8-byte aligned address. This causes an unaligned access
+// crash if the OSTime variables use 64-bit load/store instructions, which is the case in any MIPS ABI other than O32
```

Comment:
nit: do you have a reference for writing "O32" with a capital O

Reply:
It's widely inconsistent so I don't think it matters